app/models/spot.rb

```diff
@@ -1,2 +1,6 @@
 class Spot < ActiveRecord::Base
+  has_many :spot_students
+  hash_many :users, :through => :spot_students
+  belongs_to :user

```

Comment:
I don't this spots belong to users, do they? It `has_many: users` instead

Reply:
The relationship many-to-many is indicated in the `has_many :users` when the users are students. However, I give it another relationship which is one-to-many relationship if the users are the leads in `belongs_to :user`. Idk if that's neccessary.